src/Kdyby/Doctrine/ResultSet.php

```diff
@@ -360,10 +360,10 @@ public function count()
 
 
 	/**
-	 * @param ORM\Query $query
+	 * @param ORM\AbstractQuery $query
 	 * @return ResultPaginator
 	 */
-	private function createPaginatedQuery(ORM\Query $query)
+	private function createPaginatedQuery(ORM\AbstractQuery $query)
```

Comment:
Without this adjustment can not use NativeQueryBuilder
https://github.com/Kdyby/Doctrine/blob/master/src/Kdyby/Doctrine/NativeQueryBuilder.php#L76